### DISCUSSION TITLE
Problem with identical audio devices

### DISCUSSION THREAD

**Discussion by essdeeay** [Belphemur/SoundSwitch#942] [FAQ]
I have 2 x monitors of the same model/type. The sound always goes to my 2nd display. If I disconnect the 2nd display and set the playback device to be my primary display (also primary display and display #1 in Windows), I get sound from my primary display, but as soon as the secondary display is plugged back in again, sound switches to that one (and I hear nothing). My speakers are plugged into my primary display.

**Comment by essdeeay** [comment]
Actually, I've just disabled audio on my secondary display in Windows, which is fine for now - but it might be something to look at? Thanks for a great piece of software!

**Reply by Belphemur** [reply]
This should be fixed in a following version :)
